### ISSUE TITLE
Unable to load

### ISSUE THREAD

**Issue by ghost** [roxaskeyheart/Chromatics#15]
I get the following error when I try to load the Plugin:

This assembly does not have a class that implements ACT's plugin interface.

I followed the instructions and did a download/install from within ACT, v. 3.3.0 release 254.  I see this error has happened with other programs, but there's no clarity on what it means or any consistency in how to fix it.

**Comment by logicallysynced** [comment]
Can you attach a screenshot please?

**Comment by korazian** [comment]
i get the same error, screenshot https://imgur.com/AtLFNCj

**Comment by logicallysynced** [comment]
I can see ACT is trying to load "Corale.Colore.dll" which is a dependency and not the actual plugin. Try and manually add "Chromatics.dll" (which should be in the folder above the "lib" folder) from where you downloaded and unzipped the files to OR if you tried to install via ACT's "Get Plugins" you can find it under the ACT plugins folder under

**HardDrive** > ProgramData > Roaming > AdvancedCombatTracker

Or a directory very similar to that.

**Comment by korazian** [comment]
these are the only files in the lib folder https://imgur.com/KVVSlsP

**Comment by logicallysynced** [comment]
Go up one level to the Chromatics folder and you will find the "Chromatics.dll".

**Comment by logicallysynced** [comment]
Additionally, make sure you are downloading the actual release and not the repository. The repo is not up to date or structured correctly.

https://github.com/roxaskeyheart/Chromatics/releases

**Comment by korazian** [comment]
i downloaded from the link you sent, found the chromatics.dll but when trying to add/enable
http://imgur.com/hMFXHcp

**Comment by logicallysynced** [comment]
Did you unblock the zip file before extracting?

**Comment by logicallysynced** [comment]
Closing due to lack of response. Please submit a new ticket if you are still experiencing issues.
